### DISCUSSION TITLE
Have you ever used the keyboard you are developing?

### DISCUSSION THREAD

**Discussion by TheThomasAmdreson** [FossifyOrg/General-Discussion#627] [Ideas]
1st, there is no way to disable the empty line with the “clipboard” and the “gear” icon.
2nd Spaces between rows of letters are too large
3rd 4-line keyboard display with numbers in the top line, when switching to “symbols” there is no need to change to a 3-line keyboard layout.
4th language switching is a more priority goal; for people, English is not the native language. How to use emoticons
5th you don't use "layers". Well, that is, you use “emoji/change language” for one key
In the line mode "+ number row" you can put the symbols (!/1) (@/2) on the second layer, just like on a real keyboard. so that there is no need to press an extra key to enter the 4th row of symbols. You can assign additional characters to letter keys. in this case, the Character Layout will be needed only for entering the rarest characters.
6th Scrolling through emoji from top to bottom is not very convenient. Even so it is very inconvenient. You need to scroll left and right page by page. And “recent” should be on the 0th page on the left. And emoji starts 1st

**Comment by Aga-C** [comment]
Can you create feature requests for these suggestions, so they won't get lost in Discussions? But be aware that some of the items you've pointed out have been already requested.
